cpu_mitigations/states/init.sls

```diff
@@ -0,0 +1,58 @@
+# This file is part of cpu-mitiation-formula.
+#
+# Foobar is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Foobar is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Foobar.  If not, see <http://www.gnu.org/licenses/>.
+
+# This state configures cpu mitigation kernel parameters via the grub conf.
+
+{% from "cpu_mitigations/map.jinja" import map with context %}
+{%- set selected = salt['pillar.get']('mitigations:name', 'Auto') %}
+
+# check for support os version
+{%- set os_ver = grains['osrelease'] %}
+{% if '12' in os_ver and '12.2' <= os_ver %}
+{%- set grub_line_start = map.grub_line.get('12') %}
+{% elif '42.3' == os_ver %}
+{%- set grub_line_start = map.grub_line.get('12') %}
+{% elif '15' in os_ver %}
+{%- set grub_line_start = map.grub_line.get('15') %}
+{% endif %}
```

Comment:
Should we provide a default value here for `grub_line_start`? 

Because, in case `osrelease` is `12.1` or when the OS is Tumbleweed (where `osrelease` is something like `20190607`) then this formula will not set any value for `{{ grub_line_start }}` which might cause some unexpected results when the `file.replace` is executed.